domains/link.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "LinkerScript",
+    "email": "min@effobe.com"
+  },
+  "record": {
+    "CNAME": "6498c941-9031-4e57-9fc0-12716fa187f4.repl.co"
```

```suggestion
    "URL": "8m.repl.co"
```
`6498c941-9031-4e57-9fc0-12716fa187f4.repl.co` is 404 it but `m8.repl.co` is work.
and URL Redirection is better because I think it(repl.it service) can't set custom URL.
so Redirection is better

---

When I clicked the custom domain prompt it asked me to set the CNAME record as ‘6498c941-9031-4e57-9fc0-12716fa187f4.repl.co’